modules/shapefile/src/lib/parsers/parse-shapefile.js

```diff
@@ -101,6 +106,19 @@ function joinProperties(geometries, properties) {
   return features;
 }
 
+/**
```

Comment:
Sounds like this should be a `gis` module utility?

If so it could accept an options object and we could handle the options in here.

Reply:
Currently `@math.gl/proj4` isn't a `@loaders.gl/gis` dependency. Adding this function to the `gis` submodule will require adding proj4 as a dependency, and will significantly increase the `gis` bundle size.

My thinking around creating a flexible `transformCoords` function in `@loaders.gl/gis` was that this two-line function could stay out and only necessary modules (i.e. Shapefile and Flatgeobuf for now) would have the bundle size penalty.

Reply:
OK fine for now.